web/src/containers/PatientDetails/PatientDocument/index.tsx

```diff
@@ -41,10 +43,16 @@ export function PatientDocument(props: Props) {
     const { setBreadcrumbs } = useContext(PatientHeaderContext);
     const location = useLocation();
 
+    const [draftSaveState, setDraftSaveState] = useState<RemoteData>(notAsked);
```

Comment:
I would prefer to use `Response` instead of `State` in this case
```
const [draftSaveResponse, setDraftSaveResponse] = useState<RemoteData<QuestionnaireResponse>>(notAsked);
```